rmc/resources/resource_utils.py

```diff
@@ -1,7 +1,5 @@
+"""Script containing resource utility constants."""
 BUILDS = ["GRCh37", "GRCh38"]
-FLAGSHIP_LOF = "gs://gnomad-public-requester-pays/papers/2019-flagship-lof/v1.0"
-RMC_PREFIX = "gs://regional_missense_constraint"
-RESOURCE_PREFIX = "gs://regional_missense_constraint/resources"
 CURRENT_VERSION = "2.1.1"
```

Comment:
Can we rename this to `CURRENT_GNOMAD_VERSION` so as not to be confused with freezes?